### PR TITLE
Russian translation for new stage 3 proposals

### PR DESCRIPTION
**Pull request by maramigin-e**
Russian translation for new stage 3 proposals

### CODE REVIEW

File: _data/ru/stage3.yml [hunk 1]
```diff
@@ -88,16 +88,16 @@
     - Sven Sauleau
     - Dan Clark
     - Daniel Ehrenberg
-- title: JSON Modules
+- title: Модули JSON
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
- title: JSON модули
```

File: _data/ru/stage3.yml [hunk 2]
```diff
@@ -88,16 +88,16 @@
...
   example: >-
     import json from &#x22;./foo.json&#x22; assert { type: &#x22;json&#x22; };
   presented:
-    - date: January&#xA0;2021
+    - date: Январь&#xA0;2021
       url: >-
         https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-25.md#json-modules-for-stage-3
   has_specification: true
-  description: Proposal to import JSON files as modules.
+  description: Предложение импортировать файлы JSON в виде модулей.
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
  description: Импорт JSON-файлов в виде модулей.
```

File: _data/ru/stage3.yml [hunk 3]
```diff
@@ -108,7 +108,7 @@
     - Daniel Ehrenberg
     - Myles Borins
     - Dan Clark
-- title: Ergonomic Brand Checks
+- title: Эргономичные проверки бренда
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]: I think it's better to leave it in English until we find a good translation

File: _data/ru/stage3.yml [hunk 4]
```diff
@@ -120,22 +120,22 @@
       }
     }
   presented:
-    - date: January&#xA0;2021
+    - date: Январь&#xA0;2021
       url: >-
         https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-27.md#ergonomic-brand-checks
   has_specification: true
-  description: EcmaScript proposal to provide brand checks without exceptions.
+  description: Предложение EcmaScript предоставить проверку бренда без исключений.
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]: same

File: _data/ru/stage3.yml [hunk 5]
```diff
@@ -120,22 +120,22 @@
...
     - Jordan Harband
 - id: proposal-class-static-block
-  title: Class Static Block
+  title: Статический блок класса
   authors:
     - Ron Buckton
   champions:
     - Ron Buckton
-  description: Class static blocks provide a mechanism to perform additional static initialization during class definition evaluation. This is not intended as a replacement for public fields, as they provide useful information for static analysis tools and are a valid target for decorators. Rather, this is intended to augment existing use cases and enable new use cases not currently handled by that proposal.
+  description: Статические блоки класса предоставляют механизм для выполнения дополнительной статической инициализации во время оценки определения класса. Это не предназначено для замены публичных полей, поскольку они предоставляют полезную информацию для инструментов статического анализа и являются допустимой целью для декораторов. Скорее, это предназначено для расширения существующих вариантов использования и включения новых вариантов использования, которые в настоящее время не обрабатываются этим предложением.
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
  description: Статические блоки класса предоставляют механизм для выполнения дополнительной статической инициализации во время исполнения определения класса. Статические блоки не предназначены для замены публичных полей, поскольку те предоставляют полезную информацию для инструментов статического анализа и являются допустимой целью для декораторов. Блоки добавлены расширения существующих и добавления новых вариантов использования, которые возможно добавятся в будущем. 
```

File: _data/ru/stage3.yml [hunk 6]
```diff
@@ -192,7 +192,7 @@
     - Maggie Pint
     - Matt Johnson
     - Philipp Dunkel
-  description: Date has been a long-standing pain point in ECMAScript. This proposes Temporal, a global Object that acts as a top-level namespace (like Math), that brings a modern date/time API to the ECMAScript language. For a detailed breakdown of motivations, see  the proposal readme.
+  description: Date была давней проблемой в ECMAScript. Это предложение, что Temporal глобальный объект, который действует как пространство имен верхнего уровня (например, Math), который привносит современный API даты / времени в язык ECMAScript. Более детально можно посмотреть в файле readme для предложения.
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
  description: Date была давней проблемой в ECMAScript. Это предложение добавляет глобальный объект Temporal, который действует как пространство имен верхнего уровня (как Math), который привносит современный API даты / времени в язык ECMAScript. Более детально про мотивацию можно прочитать в файле readme этого предложения.
```

File: _data/ru/stage3.yml [hunk 7]
```diff
@@ -206,31 +206,31 @@
...
     if (Object.hasOwn(object, "foo")) {
       console.log("has property foo")
     }
   has_specification: true
   presented:
-    - date: "May\_2021"
+    - date: "Май\_2021"
 - id: proposal-resizablearraybuffer
-  title: In-Place Resizable and Growable ArrayBuffers
+  title: ArrayBuffers с изменяемым размером и увеличением на месте
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
  title: ArrayBuffers с изменяемым размером
```

File: _data/ru/stage3.yml [hunk 8]
```diff
@@ -206,31 +206,31 @@
...
   champions:
     - Shu-yu Guo
-  description: ArrayBuffers have enabled in-memory handling of binary data and have enjoyed great success. This proposal extends the ArrayBuffer constructors to take an additional maximum length that allows in-place growth and shrinking of buffers. Similarly, SharedArrayBuffer is extended to take an additional maximum length that allows in-place growth. The transfer method is also re-introduced here as a standard way to detach ArrayBuffers, perform zero-copy moves, and to "fix" resizable ArrayBuffer instances to ArrayBuffer instances.
+  description: ArrayBuffers позволили обрабатывать двоичные данные в памяти и пользуются большим успехом. Это предложение расширяет конструкторы ArrayBuffer на дополнительную максимальную длину, позволяющую увеличивать и уменьшать буферы на месте. Точно так же SharedArrayBuffer расширяется, чтобы получить дополнительную максимальную длину, которая допускает рост на месте. Метод передачи также повторно представлен здесь как стандартный способ отсоединения ArrayBuffers, выполнения перемещений с нулевым копированием и «исправления» экземпляров ArrayBuffer с изменяемым размером для экземпляров ArrayBuffer.
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]:
```suggestion
  description: ArrayBuffers позволили обрабатывать двоичные данные в памяти и пользуются большим успехом. Это предложение расширяет конструкторы ArrayBuffer, чтобы принимать дополнительную максимальную длину, что позволит увеличивать и уменьшать размер буфера. Также предлагается расширить конструктор SharedArrayBuffer. Метод tranfser также представлен здесь как стандартный способ ArrayBuffers, выполнения операций без копирования и приведения экземпляров ArrayBuffer с изменяемым размером к экземплярам ArrayBuffer. 
```

File: _data/ru/stage3.yml [hunk 9]
```diff
@@ -162,26 +162,26 @@
...
-  title: Error Cause
+  title: Причина Ошибки
   authors:
     - Chengzhong Wu
   champions:
     - Ron Buckton
     - Chengzhong Wu
     - Hemanth HM
-  description: Errors will be constructed to represent runtime abnormalities. To help unexpected behavior diagnosis, errors need to be augmented with contextual information like error messages, error instance properties to explain what happened at the time.  If the error were thrown from deep internal methods, the thrown error may not be straightforward to be easily conducted without proper exception design pattern. This proposal introduces a convenient way to forward the cause.
+  description: Ошибки будут созданы для представления отклонений во время выполнения кода. Чтобы помочь в диагностике неожиданного поведения, ошибки необходимо дополнить контекстной информацией, такой как сообщения об ошибках, свойства экземпляра ошибки, чтобы объяснить, что произошло в то время. Если ошибка возникла из-за глубоких внутренних методов, возникшую ошибку может быть непросто устранить без надлежащего шаблона проектирования исключений. Это предложение представляет удобный способ представления причины ошибки
```
Comment (chicoxyzzy) [inline _data/ru/stage3.yml]: We need a better wording here. I don't have ideas right now, I think it's better to leave it in English for now

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

**Comment by chicoxyzzy** [comment]
Also please rename issue to "Russian translation for new stage 3 proposals". Thanks!

File: _data/ru/stage3.yml (hunk 3 above)
Comment (myshov) [reply _data/ru/stage3.yml]: Maybe "Эргономичные бренд-проверки" or with an anglicism "Эргономичные бренд-чеки"?

**Comment by myshov** [comment]
I want to add my 5 cents. I think it would be great if every title of proposal would have russian translation with original english title in parentheses

File: _data/ru/stage3.yml (hunk 5 above)
Comment (alexkuz) [reply _data/ru/stage3.yml]: (by "that proposal" they mean https://github.com/tc39/proposal-class-fields I believe, so it's not about some new proposals in future)

```suggestion
  description: Статические блоки класса предоставляют механизм для дополнительной статической инициализации во время исполнения определения класса. Статические блоки не заменяют публичные поля, поскольку те совместимы с декораторами и могут быть полезны для инструментов статического анализа. Они скорее дополняют существующий синтаксис, а также дают новые варианты использования, не предусмотренные полями класса. 
```
, something like that

**Comment by maramigin-e** [comment]
I understand correctly that those sentences that were not marked here, but I translated - is everything correct there? for example temporal.

If possible, you can give a little more time, I will try to re-translate the errors.

**Comment by chicoxyzzy** [comment]
> I understand correctly that those sentences that were not marked here, but I translated - is everything correct there? for example temporal.

That's correct. You don't need to change those lines.

> If possible, you can give a little more time, I will try to re-translate the errors.

Sure! No problem!

**Comment by chicoxyzzy** [comment]
> I want to add my 5 cents. I think it would be great if every title of proposal would have russian translation with original english title in parentheses

I don't think we need this for most of proposals

**Comment by myshov** [comment]
> I don't think we need this for most of proposals

I meant the next thing. How does people could match the translated name of proposal with the original one, if they need to google more information on a given proposal?

**Comment by maramigin-e** [comment]
Hey, @chicoxyzzy !

I added new commit message.

If possible, I would like to contribute further)

**Comment by codehag** [comment]
Are we ready to merge? cc @chicoxyzzy

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
I've seen that some of last fixes do not follow review comments suggestions, but I didn't have time to check carefully yet. I'll try to review tomorrow.

**Comment by chicoxyzzy** [comment]
Hi @maramigin-e. We've migrated website to 11ty engine recently so this PR needs some updates. Do you mind to update this PR or (and I think that would be easier way) create new one? Also I see that not all suggestions was applied, but I think we can just make a new iteration anyway.

**Comment by chicoxyzzy** [comment]
I think this PR could be closed as it has too many conflicts. Please feel free to open new one.

**Pull request closed without merging**